SelfContained/SelfContained.csproj

```diff
@@ -12,14 +12,9 @@
     <DebugType>embedded</DebugType>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <ProjectReference Include="..\Terminal.Gui\Terminal.Gui.csproj" />
-    <TrimmerRootAssembly Include="Terminal.Gui" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
`SelfContained` also needs to use the local nuget package in release mode, because some of the trimmed feature handles in deferent way in debug and release mode.

Reply:
I've removed all conditions, that means all configurations regardless will use the same package.  Isn't that what we want?

Reply:
Does we can debug if using the release package? If we can that fine, otherwise I prefer to use the project reference.

Reply:
> I've removed all conditions, that means all configurations regardless will use the same package. Isn't that what we want?

But you are referencing the project and not the package. In the `SelfContained` project it need at least in the release mode, reference the package because sometimes e may failing when running in release mode. That is important.